components/rhs_root_post/rhs_root_post.jsx

```diff
@@ -229,25 +229,13 @@ class RhsRootPost extends React.PureComponent {
     };
 
     render() {
-        const {post, isReadOnly, teamId, channelIsArchived, channelType, channelDisplayName} = this.props;
+        const {post, isReadOnly, teamId, channelIsArchived} = this.props;
```

Comment:
Since we're no longer using `channelType` or `channelDisplayName`, can you remove both of those from the props and make sure they aren't passed in from `mapStateToProps` as well?

Reply:
Will do.